components/dashboard/src/projects/Prebuilds.tsx

```diff
@@ -280,8 +288,16 @@ export default function (props: { project?: Project; isAdminDashboard?: boolean
                                                         alt={p.info.changeAuthor}
                                                     />
                                                 )}
-                                                Authored {formatDate(p.info.changeDate)} ·{" "}
-                                                {p.info.changeHash?.substring(0, 8)}
+
+                                                <Tooltip
+                                                    className="w-fit"
+                                                    content={`Authored On ${dayjs(p.info.changeDate)
+                                                        .toDate()
+                                                        .toLocaleString("hi-IN")}`}
```

Comment:
**suggestion**: Could we use the following guideline for which format to use when:

> If possible to detect browser settings, let's display the localized time (`Jan 18, 2023`). Otherwise, let's fall back to the date using the [`ISO 8601`](https://www.iso.org/iso-8601-date-and-time-format.html) format `YYYY-MM-DD` and append the time optionally at the end.

```
// Localized time
Jan 18, 2023

// ISO 8601
YYYY-MM-DD HH:MM:SS
```

Besides that we should try to always display date and time information in the absolute format when possible but we can make this change in a later iteration. 🛹